west-european-politics.csl

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>West European Politics</title>
+    <id>http://www.zotero.org/styles/west-european-politics</id>
+    <link href="http://www.zotero.org/styles/west-european-politics" rel="self"/>
```

Comment:
did you base this on any existing style? If so which (we like to include a link with `rel="template"`